PatternPal/PatternPal/Services/PatternPalService.cs

```diff
@@ -29,10 +30,10 @@ public class PatternPalService : Protos.PatternPal.PatternPalBase
             //TODO programming language is not compatible add a feedback message for project check for .csproj?
         }
 
-        List<string> files;
+        List< string > files;
         // Return all the .cs files (and in all subdirectories)
         // Does not include files part of project but not in directory
-        if (string.IsNullOrEmpty(pathProject))
+        if (!string.IsNullOrEmpty(pathProject))
```

Comment:
Dit klinkt raar aangezien de inhoud van de if niet gewijzigd is.

Reply:
Wat klinkt hier raar aan? Dit was een bug geintroduceerd door Daan die ik direct heb gefixt.